docusaurus/sidebars.js

```diff
@@ -573,7 +573,7 @@ const sidebars = {
           label: 'Collaboration',
           id: 'cloud/projects/collaboration',
           customProps: {
-            new: true,
+            new: false,
```

Comment:
This works perfectly, but to lighten the sidebar file, we might as well delete lines 575-577 :) And to further shorten the declaration of this TOC entry, you could simply replace the whole block of lines 571-577 with just `'cloud/projects/collaboration'`. 
If the label of the TOC item is not "Collaboration", you can explicitly define it as such using [`sidebar_label`](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#sidebar_label) in the corresponding Markdown file.

Just a tip, but again everything will work perfectly fine here :)

Reply:
I have to admit, I might have chosen the lazy and messy way to do it, I'm sorry 😂  But since you've pointed it out I fixed it 🤓

Reply:
No worries :D As I wrote, it would work perfectly, and actually, your approach is even better for the "Updated" badge, as it allows us to turn the badge on and off quickly. 🚀

For "new", though, I'd argue that once it's not new, it won't become new again 😄